tests/backstop.json

```diff
@@ -88,23 +88,28 @@
 			"referenceUrl": "https://db-ui.github.io/core/patterns/elements-image/index.html"
 		},
 		{
-			"label": "Elements – Input",
-			"url": "http://127.0.0.1:8080/patterns/elements-input/index.html",
+			"label": "Elements – Input Sizes",
+			"url": "http://127.0.0.1:8080/patterns/elements-input-input-sizes/elements-input-input-sizes.rendered.html",
+			"referenceUrl": "https://db-ui.github.io/core/patterns/elements-input/index.html"
+		},
+		{
+			"label": "Elements – Input Variations",
+			"url": "http://127.0.0.1:8080/patterns/elements-input-input-variations/elements-input-input-variations.rendered.html",
```

Comment:
let's not include the different pseudo-patterns, but only test the general page from now on to reduce the test cases, meaning to revert adding the other pages to this file, BUT we would still need to change selectors for the interactions as you did for "Elements – Input - focused" (and we need it for "Elements – Input / hovered" as well).

Reply:
done